src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,14 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and overriding the clone method is not a common practice for entities in Hibernate. Entities are usually managed by the session and their lifecycle is well-defined. Cloning entities can lead to unexpected behaviors, especially when dealing with managed entities and their relationships. If you need to create a copy of an entity, consider using copy constructors or a dedicated method for copying properties.